src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -41,9 +41,12 @@
 #include <seastar/coroutine/parallel_for_each.hh>
 #include <seastar/http/api_docs.hh>
 #include <seastar/http/exception.hh>
+#include <seastar/util/log.hh>
 
 namespace pandaproxy::schema_registry {
 
+ss::logger srlog{"schema_registry"};
```

Comment:
nit: It would be good to use clearly distinct logger names for HTTP access logs vs general logging.

This is already the case (actual debug logs use the `pandaproxy` logger), but It's not very obvious and I suspect folks writing schema registry code in future might use this logger thinking it's a general debug log rather than the request log.

Maybe call them "schema_registry_access" and "rest_proxy_access" to distinguish from any other general logs?

Reply:
We may also want to rename the `pandaproxy` logger to something more general such as `httplog` or something. This is just a discussion point though and does not need to be in this PR.